domain-models-runtime/src/test/java/org/folio/rest/persist/PgUtilIT.java

```diff
@@ -1107,6 +1107,49 @@ public void optimizedSQLwithNo500(TestContext testContext) {
         }));
   }
 
+   @Test
+  public void deleteByCQLwithNo500(TestContext testContext) {
+    PostgresClient pg = PostgresClient.getInstance(vertx, "testtenant");
+    insert(testContext, pg, "delete_test",  1);
+    PgUtil.delete("users",  "username=delete_test",
+        okapiHeaders, vertx.getOrCreateContext(), ResponseWithout500.class, testContext.asyncAssertFailure(e -> {
+          assertThat(e, is(instanceOf(NullPointerException.class)));
+        }));
+  }
+
+  @Test
+  public void deleteByCQLwithNo204(TestContext testContext) {
+    String uuid = randomUuid();
+    post(testContext, "Pippilotta", uuid, 201);
+
+    PgUtil.delete("users", "username=delete_test",
+        okapiHeaders, vertx.getOrCreateContext(),
+        Users.PutUsersByUserIdResponse.class,
+        asyncAssertSuccess(testContext, 204));
```

Comment:
This only checks that the response code is 204.
It should also check that PgUtil.delete correctly deletes the records filtered by CQL.
The test case should start with at least 3 records and use a CQL that deletes at least 2 records but not all records.